src/jit/codegenlinear.cpp

```diff
@@ -1507,71 +1502,28 @@ void CodeGen::genSetBlockSrc(GenTreeBlk* blkNode, regNumber srcReg)
 
 void CodeGen::genConsumeBlockOp(GenTreeBlk* blkNode, regNumber dstReg, regNumber srcReg, regNumber sizeReg)
 {
-    // We have to consume the registers, and perform any copies, in the actual execution order.
-    // The nominal order is: dst, src, size.  However this may have been changed
-    // with reverse flags on the blkNode and the setting of gtEvalSizeFirst in the case of a dynamic
-    // block size.
+    // We have to consume the registers, and perform any copies, in the actual execution order: dst, src, size.
```

Comment:
This is not strictly correct. I think it should be something like: "We have to consume the registers, and perform any copies, in the canonical operand order (dst, src, size), as that is the order used also by the register allocator."
Then, the comment below should also be changed to use "canonical order" (or whatever term is used) instead of "execution order".